models/contact/src/index.ts

```diff
@@ -69,7 +70,7 @@ export class TContact extends TDoc implements Contact {
   @Index(IndexKind.FullText)
   name!: string
 
-  avatar?: string | null
+  avatar?: Avatar
```

Comment:
I'm against having a structure here, since all import code like, our chrome plugin need to be updated, migrated. My initial proposal was to use {schema://uri} to to be handled by different providers. So default one without schema will be image attached to our minio, http:// https:// schema will be icon from internet. gravatar:// will be for gravatar and color:// will be for color. Could we archive and make compatible with just pure string| null type?